x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/waterfall_chart.tsx

```diff
@@ -86,7 +86,6 @@ export const WaterfallChart = ({
   barStyleAccessor,
   renderSidebarItem,
   renderLegendItem,
-  maxHeight = 800,
 }: WaterfallChartProps) => {
```

Comment:
I think keeping the maxHeight prop could be beneficial if we port this component to be used in other views outside of the Uptime app. I *think* I heard talk about that a few days ago, but I can no longer find it. If we keep the maxHeight prop, but without a default, it should keep things flexible for other uses, while still enabling us to omit a maxHeight for our use case.

Reply:
i have adjusted the approch